.ci-build-without-test.sh

```diff
@@ -23,7 +23,7 @@ export AFU="${AFU:-../annotation-tools/annotation-file-utilities}"
 # Don't use `AT=${AFU}/..` which causes a git failure.
 AT=$(dirname "${AFU}")
 
-## Build annotation-tools (Annotation File Utilities)
+# Build annotation-tools (Annotation File Utilities)
```

Comment:
Not sure why this is changed... if you want it consistent, there is another `##` below.

Reply:
Done